lib/cli/commands/orders/marketorder.ts

```diff
@@ -0,0 +1,22 @@
+import { Arguments } from 'yargs';
+import * as limitorder from './limitorder';
+
+export const command = 'marketorder <pair_id> <order_id> <quantity>';
+
+export const describe = 'place a market order';
+
+export const builder = {
+  pair_id: {
+    type: 'string',
+  },
+  order_id: {
+    type: 'string',
+  },
+  quantity: {
+    type: 'number',
+  },
+};
+
+export const handler = (argv: Arguments) => {
+  limitorder.handler(argv);
```

Comment:
Do we need to set a 0 price here like `{ ...argv, price: 0 }` or does it work as is? When we do `setPrice` in the limitorder file we set an undefined value there instead of a number. I'm guessing it's probably fine, I just haven't tested it. If it works for you, then we can keep as is.

Reply:
It is working right now because the gRPC server receives the value 0 for `price` even if it set to `undefined` on the client side.

Reply:
That's what I thought, works for me.